spec/unit/x509/cert_provider_spec.rb

```diff
@@ -19,10 +19,11 @@ def expects_public_file(path)
       current_sid = Puppet::Util::Windows::SID.name_to_sid(Puppet::Util::Windows::ADSI::User.current_user_name)
       sd = Puppet::Util::Windows::Security.get_security_descriptor(path)
       expect(sd.dacl).to contain_exactly(
-        an_object_having_attributes(sid: Puppet::Util::Windows::SID::LocalSystem, mask: 0x1f01ff),
-        an_object_having_attributes(sid: Puppet::Util::Windows::SID::BuiltinAdministrators, mask: 0x1f01ff),
-        an_object_having_attributes(sid: current_sid, mask: 0x1f01ff),
-        an_object_having_attributes(sid: Puppet::Util::Windows::SID::BuiltinUsers, mask: 0x120089)
+        an_object_having_attributes(sid: Puppet::Util::Windows::SID::LocalSystem, mask: 0x12019f),
+        an_object_having_attributes(sid: Puppet::Util::Windows::SID::BuiltinAdministrators, mask: 0x120089),
+        an_object_having_attributes(sid: current_sid, mask: 0x12019f),
+        an_object_having_attributes(sid: Puppet::Util::Windows::SID::BuiltinUsers, mask: 0x120089),
+        an_object_having_attributes(sid: Puppet::Util::Windows::SID::Everyone, mask: 0x120089)
```

Comment:
Not sure if this set of permissions are right. addition to the ones before: `sid: Puppet::Util::Windows::SID::Everyone`